jcl/src/openj9.jvm/share/classes/com/ibm/jvm/CRIUDumpPermission.java

```diff
@@ -0,0 +1,50 @@
+/*[INCLUDE-IF CRIU_SUPPORT]*/
+/*******************************************************************************
+ * Copyright (c) 2021, 2021 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package com.ibm.jvm;
+
+import java.security.BasicPermission;
+
+/**
+ * The permission class for operations on the com.ibm.jvm.CRIUSupport class.
+ * Allowing code access to this permission will allow applications to trigger 
+ * CRIU dumps
+ * 
+ * A dump file will be a complete image of the applications memory. Code with read
+ * access to dump files produced by com.ibm.jvm.CRIUSupport should be considered as having
+ * access to any information that was within the application at the time the dump
+ * was taken. 
+ */
+public class CRIUDumpPermission extends BasicPermission {
+
+	/**
+	 * Serial version UID
+	 */
+	private static final long serialVersionUID = 1L;
```

Comment:
Is this really what `serialver` generated for this class?

Reply:
Yes, that was the eclipse default

Reply:
I manually ran `serialver` and generated a new UID